modules/installation/pages/uyuni-server-setup.adoc

```diff
@@ -0,0 +1,171 @@
+{productname} can sometimes send a large volume of notification emails.
+You can disable email notifications in the {webui} after setup, if you need to.
+
+. Enter your certificate information and a password.
+Passwords must be at at least seven characters in length, and must not contain spaces, single or double quotation marks (``'`` or ``"``), exclamation marks (``!``), or dollar signs (``$``).
+Always store your passwords in a secure location.
++
+
+[IMPORTANT]
+.Certificate Password
+====
+Without this password it will not be possible to set up a {productname} Proxy Server.
+====
+
+. Click btn:[Next] to continue.
++
+
+// image::quickstart-mgr-setup4-ncurses.png[scaledwidth=80%]
+
++
+// To be verified
+. From the menu:Uyuni Setup[Database Settings] screen, enter a database user and password and click btn:[Next] to continue.
+Passwords must be at at least seven characters in length, and must not contain spaces, single or double quotation marks (``'`` or ``"``), exclamation marks (``!``), or dollar signs (``$``).
+Always store your passwords in a secure location.
++
+
+// image::quickstart-mgr-setup5-ncurses.png[scaledwidth=80%]
+
+. Click btn:[Next] to continue.
+
+. Click btn:[Yes] to run setup when prompted.
+
+. When setup is complete, click btn:[Next] to continue.
+You will see the address of the {productname} {webui}.
+
+. Click btn:[Finish] to complete {productname} setup.
+
+// In the next section you will create the administrator's account and synchronize with {scc}.
+
+
+== Creating the Main Administration Account
+
+This section will walk you through creating your organization's main administration account for {productname}.
+
+[WARNING]
+.Admin and User Accounts
+====
+The main administration account is the _highest authority account_ within {productname} and therefore account access information should be stored in a secure location.
+
+For security it is recommended that the main administrator creates _low level admin accounts_ designated for administration of organizations and individual groups.
+====
+
+
+.Procedure: Setting Up the Main Administration Account
+. In the browser, enter the address provided after completing setup.
+With this address you open the {productname} {webui}.
+
+. In the {webui}, navigate to the menu:Create Organization[Organization Name] field and enter your organization name.
+
+. In the menu:Create Organization[Desired Login] and menu:Create Organization[Desired Password] fields, enter your username and password.
+
+. Fill in the Account Information fields including an email for system notifications.
+
+. Click btn:[Create Organization] to finish creating your administration account.
++
+
+// image::quickstart-mgr-setup-admin1.png[scaledwidth=80%]
+
+You are now presented with the {productname} menu:Home[Overview] page.
+// In the next section you will prepare the server for connecting the first client.
+
+
+== Synchronizing Products from {scc}
```

Comment:
We need to clearly state that this optional.

Reply:
ok, I added such tags.